src/databricks/labs/ucx/source_code/jobs.py

```diff
@@ -241,6 +246,7 @@ def _register_pipeline_task(self, graph: DependencyGraph):
                 return
             if library.notebook.path:
                 notebook_path = library.notebook.path
+                # TODO: Support DBFS here.
```

Comment:
not here either

Reply:
though, we're missing wheel

Reply:
@nfx: Can you elaborate on this? I don't fully understand, but think wheels are covered?